Dockerfile

```diff
@@ -0,0 +1,5 @@
+# use node as base image
+FROM ryanthemanuel/terra-node:test_parallel
```

Comment:
Where can I find this?

Reply:
It gets created as part of the terra-toolkit build.  It's in the docker directory